migrations/20170818-origin-rename.sql

```diff
@@ -0,0 +1,9 @@
+/* Add copyright header here.
```

Comment:
This is a trivial one-line file, so I'm not sure the copyright notice really matters, but just to model good behavior for others, let's put one in.